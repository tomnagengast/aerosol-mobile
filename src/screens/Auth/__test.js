/* eslint-disable */
import Connected, { Auth } from './';

describe('Component', () => {
    let wrapper = shallow(<Auth/>);

    it('renders something', () => {
        expect(wrapper.length).toEqual(1);
        expect(wrapper).toMatchSnapshot();
    });
});
