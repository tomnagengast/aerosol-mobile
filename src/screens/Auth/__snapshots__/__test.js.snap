// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders something 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 20,
      "width": 750,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Template Screen
  </Text>
  <Button
    title="Template Button"
  />
</View>
`;
